### PR TITLE
[rb] get specs passing on windows

### PR DESCRIPTION
**Pull request by titusfortner**
This gets local specs passing on Windows (build 16257) with production Firefox and latest Chrome, IE & Edge browsers and drivers.

The Edge stuff might be better otherwise implemented, but this gets it working. It needs to use the OSS commands for actions, but the way I have it toggling between OSS & W3C has been to override at `#execute` in the `COMMAND` hash, and all the interactions methods call the Bridge methods directly.

I can't get the remote specs working on Windows with current code to verify Ruby specs are passing there. :-/

---
This change is [Reviewable](https://reviewable.io/reviews/seleniumhq/selenium/4484)

### CODE REVIEW

File: rb/spec/integration/selenium/webdriver/element_spec.rb [hunk 1]
```diff
@@ -145,39 +145,41 @@ module WebDriver
...
 
-      context 'size and location' do
-        it 'should get current location' do
-          driver.navigate.to url_for('xhtmlTest.html')
-          loc = driver.find_element(class: 'header').location
-
-          expect(loc.x).to be >= 1
-          expect(loc.y).to be >= 1
-        end
+      not_compliant_on browser: :ie do
```
Comment (p0deje) [inline rb/spec/integration/selenium/webdriver/element_spec.rb]: Any tip on why it's failing?

Comment (titusfortner) [inline rb/spec/integration/selenium/webdriver/element_spec.rb]: #4480 - Jim thinks it has to do with a current issue in the atoms. It'll probably get fixed quickly enough.

**Comment by p0deje** [comment]
I've just pushed a commit which guards some of the failing specs for Edge. Is there any way you could rebase and see how it looks now? I guess the changes you did with Edge bridge should unguard some of the tests.

See 041c8159370bc9bb6b330c668a59ec9a3a777d38.

**Comment by p0deje** [comment]
Actually, I'm just going to take the changes you did for Edge and cherry-pick them...

**Comment by p0deje** [comment]
Hrm, it didn't turn out to be so simple, so I guess I'll leave that for you 😄

**Comment by tf-prbuild** [comment]
Oh shoot, did I really never merge this? Will do.

**Comment by p0deje** [comment]
I guess things have changed a lot since you last looked at Edge, is this still reasonable?

**Comment by luke-hill** [comment]
As an update when I did some local selenium testing I did get edge working and passing some specs. I can't remember how many, but a few of them did run.

**Pull request closed without merging**